boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -12,6 +12,7 @@ extern "C"
 #include "configs/App_FlowRateThresholds.h"
 #include "configs/App_WheelSpeedThresholds.h"
 #include "configs/App_SteeringAngleThresholds.h"
+#include "configs/App_BrakePressureSensorThresholds.h"
```

Comment:
Would be more consistent with the other headers
```suggestion
#include "configs/App_BrakePressureThresholds.h"
```